lib/dependencies/HarmonyAcceptDependency.js

```diff
@@ -2,31 +2,41 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var NullDependency = require("./NullDependency");
-var HarmonyImportDependency = require("./HarmonyImportDependency");
+"use strict";
+const NullDependency = require("./NullDependency");
+const HarmonyImportDependency = require("./HarmonyImportDependency");
 
-function HarmonyAcceptDependency(range, dependencies, hasCallback) {
-	NullDependency.call(this);
-	this.range = range;
-	this.dependencies = dependencies;
-	this.hasCallback = hasCallback;
+class HarmonyAcceptDependency extends NullDependency {
+	constructor(range, dependencies, hasCallback) {
+		super();
+		this.range = range;
+		this.dependencies = dependencies;
+		this.hasCallback = hasCallback;
+	}
+
+	get type() {
+		return "accepted harmony modules";
+	}
 }
-module.exports = HarmonyAcceptDependency;
 
-HarmonyAcceptDependency.prototype = Object.create(NullDependency.prototype);
-HarmonyAcceptDependency.prototype.constructor = HarmonyAcceptDependency;
-HarmonyAcceptDependency.prototype.type = "accepted harmony modules";
+HarmonyAcceptDependency.Template = class HarmonyAcceptDependencyTemplate {
+	apply(dep, source, outputOptions, requestShortener) {
+		const content = dep.dependencies
+			.map(dependency => HarmonyImportDependency.makeStatement(
+				false,
+				dependency,
+				outputOptions,
+				requestShortener
+			)).join("");
 
-HarmonyAcceptDependency.Template = function HarmonyAcceptDependencyTemplate() {};
+		if(dep.hasCallback) {
+			source.insert(dep.range[0], `function(__WEBPACK_OUTDATED_DEPENDENCIES__) { ${content}(`);
+			source.insert(dep.range[1], ")(__WEBPACK_OUTDATED_DEPENDENCIES__); }");
+			return;
+		}
 
-HarmonyAcceptDependency.Template.prototype.apply = function(dep, source, outputOptions, requestShortener) {
-	var content = dep.dependencies.map(function(d) {
-		return HarmonyImportDependency.makeStatement(false, d, outputOptions, requestShortener);
-	}).join("");
-	if(dep.hasCallback) {
-		source.insert(dep.range[0], "function(__WEBPACK_OUTDATED_DEPENDENCIES__) { " + content + "(");
-		source.insert(dep.range[1], ")(__WEBPACK_OUTDATED_DEPENDENCIES__); }");
-	} else {
-		source.insert(dep.range[1] - 0.5, ", function() { " + content + "}");
+		source.insert(dep.range[1] - 0.5, `, function() { ${content}}`);
```

Comment:
@timse I mean changing this line to
```js
source.insert(dep.range[1] - 0.5, `, function() { ${content} }`)
```
notice space before function's body closing bracket